packages/ui/Scrollable/ScrollableNative.js

```diff
@@ -728,7 +728,13 @@ class ScrollableBaseNative extends Component {
 	onScroll = (ev) => {
 		const {scrollLeft, scrollTop} = ev.target;
 
-		this.startScrollJob.startRaf(scrollLeft, scrollTop);
+		if (!this.scrollTicking) {
+			window.requestAnimationFrame(() => {
```

Comment:
This would be hard to reproduce but, for safety, should we cache the RAF id so we can cancel during `componentDidUnmount`? If so, we could probably repurpose `this.scrollTicking` to be the id holder.

Reply:
sure